compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -3467,6 +3470,9 @@ public BLangNode transform(EnumDeclarationNode enumDeclarationNode) {
         BLangUnionTypeNode bLangUnionTypeNode = (BLangUnionTypeNode) TreeBuilder.createUnionTypeNode();
         bLangUnionTypeNode.pos = bLangTypeDefinition.pos;
         for (Node member : enumDeclarationNode.enumMemberList()) {
+            if (((EnumMemberNode) member).identifier().isMissing()) {
```

Comment:
`((EnumMemberNode) member).identifier()` can extract to a separate varaible?
Same used in L3476